app/views/dashboards/show.html.erb

```diff
@@ -1,66 +1,150 @@
 <% @title = t('.title') %>
 
-<div class="l-overflow">
-  <div class="subscribed l-colspan--l colspan--l--has-border">
-    <h3 class="gems__gem__name">
-      <%= t '.latest' %>:
-    </h3>
-    <%= link_to dashboard_path(:api_key => current_user.api_key, :format => :atom), :id => 'feed', :title => t('.latest_title'), :class => 'gem__link t-list__item', 'data-icon' => '#' do %>
-      <span class="t-hidden">RSS</span>
+<!-- Subscriptions -->
+
+<div class="flex flex-wrap space-y-4 lg:space-y-0 lg:space-x-20">
```

Comment:
Any particular reason we're using flex instead of [css grid](https://tailwindcss.com/docs/grid-template-columns)?

Reply:
I found it easier to flow between 1 and 2 columns when it's just two things that either fit two across or don't.